packages/bpk-docs/src/pages/HomePage/HomePage.js

```diff
@@ -46,6 +40,9 @@ import LondonHeroImage from '../../static/london_hero.jpg';
 import HongKongHeroImage from '../../static/hongKong_hero.jpg';
 import DohaHeroImage from '../../static/doha_hero.jpg';
 
+import UpdatedAt from './UpdatedAt';
+import STYLES from './home-page.scss';
```

Comment:
FYI @sogame, this change broke some layout in the backpack homepage 😅 (see the fix here https://github.com/Skyscanner/backpack/pull/1322).

It's a bit of a gotcha that I wan't to add to the readme of `eslint-config-skyscanner` - however I reckon it actually highlights some bad practices in terms of component concerns anyway.

Reply:
Oh! Sorry about that @matthewdavidson 😓
Good point about adding a note in the readme (and maybe also in the changelog, as users upgrading might only check the changelog?)

Silver lining for me: even Backpack do this kind of mistakes 😝

Reply:
It's not your fault!! 😂 Just a heads up that this kind of thing _can_ happen.